fabric-item-api-v1/src/main/resources/fabric-item-api-v1.mixins.json

```diff
@@ -4,10 +4,19 @@
   "compatibilityLevel": "JAVA_8",
   "mixins": [
     "ItemMixin",
-    "MobEntityMixin"
+    "MobEntityMixin",
+	"CrossbowItemMixin",
```

Comment:
Keep consistent spacing with current file I guess. We need to look into enforcing editor config here and or checkstyle for json files.